platforms/r3-corda-ent/charts/auth/templates/deployment.yaml

```diff
@@ -128,9 +142,43 @@ spec:
           command: ["/bin/sh", "-c"]
           args:
           - |-
-            echo "CENM: starting CENM Auth service ..."
-            # [TODO] : initial-user-password to be fetched from vault 
-            java -jar bin/accounts-application.jar --config-file authservice.conf --initial-user-name admin --initial-user-password p4ssWord --keep-running --verbose
+            mkdir -p {{ .Values.config.volume.baseDir }}/config;
+
+            install {{ .Values.config.volume.baseDir }}/authservice.conf {{ .Values.config.volume.baseDir }}/config;
+
+            #replacement of the variables in the authservice conf file
+            export AUTH_SSL=$(cat {{ .Values.config.volume.baseDir }}/DATA/ssl/authssl)
+            sed -i -e "s*AUTH_SSL*${AUTH_SSL}*g" {{ .Values.config.volume.baseDir }}/config/authservice.conf
+
+            export SSL_TRUSTSTORE=$(cat {{ .Values.config.volume.baseDir }}/DATA/ssl/sslts)
+            sed -i -e "s*SSL_TRUSTSTORE*${SSL_TRUSTSTORE}*g" {{ .Values.config.volume.baseDir }}/config/authservice.conf
+
+            #running the auth service
+            /bin/sh
+            # main run
+            if [ -f bin/accounts-application.jar ]
+            then
+                sha256sum bin/accounts-application.jar 
+                echo
+                echo "CENM: starting CENM Auth service ..."
+                echo
+                java -Xmx{{ .Values.config.cordaJar.memorySize }}{{ .Values.config.cordaJar.unit }} -jar bin/accounts-application.jar --config-file {{ .Values.config.volume.baseDir }}/config/authservice.conf --initial-user-name admin --initial-user-password p4ssWord --keep-running --verbose
```

Comment:
can you make the initial username and password fetch from vault as well. Don't have to do for other users, just the admin user n password have it from vault

Reply:
fixed